.travis.yml

```diff
@@ -21,6 +21,9 @@ before_script:
    - sudo cp /etc/pki/ciao/cert-Scheduler-localhost.pem /etc/pki/ciao/server.pem
    - sudo cp /etc/pki/ciao/cert-CNAgent-localhost.pem /etc/pki/ciao/client.pem
    - sudo cp /etc/pki/ciao/cert-Controller-localhost.pem /etc/pki/ciao/cert-client-localhost.pem
+   - sudo openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /etc/pki/ciao/ciao-controller-key.pem -out /etc/pki/ciao/ciao-controller-cacert.pem -subj '/CN=localhost/O=CIAO test server/C=US'
```

Comment:
except that the cacert in /etc/pki/ciao is used by the compute API server, and the default location for that cert is /etc/pki/ciao/ciao-controller-cacert.pem.  The reason we move it into ubuntu's ca-trust is so that the http get requests that are in the compute_test unit tests will have the cert trusted.